server/service/client_sessions.go

```diff
@@ -4,11 +4,13 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"runtime/debug"
 )
 
 // Login attempts to login to the current Fleet instance. If login is successful,
 // an auth token is returned.
 func (c *Client) Login(email, password string) (string, error) {
+	debug.PrintStack()
```

Comment:
Do we need this? (Not sure about dumping the whole stack to `fleetctl` users...)

Reply:
Good catch. Yes, I was troubleshooting something and needed to see how something was being called without using dlv